model/pipeline/src/main/proto/metrics.proto

```diff
@@ -62,8 +62,13 @@ message MonitoringInfoSpecs {
     // TODO(BEAM-6926): Add the PTRANSFORM name as a required label after
     // upgrading the python SDK.
     USER_COUNTER = 0 [(monitoring_info_spec) = {
-      urn: "beam:metric:user:",
+      urn: "beam:metric:user",
```

Comment:
it may be too late now, but does it make sense to call this `"beam:metric:user_counter"`?